src/components/Camera/index.tsx

```diff
@@ -0,0 +1,115 @@
+import { h, FunctionComponent } from 'preact'
+import Webcam, { WebcamProps } from 'react-webcam-onfido'
+import classNames from 'classnames'
+
+import { localised } from '../../locales'
+import withPermissionsFlow from '../CameraPermissions/withPermissionsFlow'
+import CameraButton from '../Button/CameraButton'
+
+import withFailureHandling from './withFailureHandling'
+import style from './style.scss'
+
+import type { CameraProps } from '~types/camera'
+import type {
+  WithLocalisedProps,
+  WithFailureHandlingProps,
+  WithTrackingProps,
+  WithPermissionsFlowProps,
+} from '~types/hocs'
+
+// Specify just a camera width (no height) because on safari if you specify both
+// height and width you will hit an OverconstrainedError if the camera does not
+// support the precise resolution.
+// Using width here because on some special devices (e.g. Samsung Galaxy),
+// setting 720px in height results to 720x960 resolution instead of the desired 720x1280.
+const DEFAULT_CAMERA_WIDTH_IN_PX = 1280 // HD 720p
```

Comment:
Okay so does it mean declaring `WIDTH` instead of `HEIGHT` will break on some Android devices?

Reply:
Ah, sorry I didn't notice that we were setting height before and now we are setting width. I'm not sure if it would break, hopefully not! We would have to instruct Applause to run the tests on multiple devices.